src/Graphics/TextRenderer.cpp

```diff
@@ -0,0 +1,395 @@
+/******************************************************************************
+ * Copyright (c) 2018-2024 openblack developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/openblack/openblack
+ *
+ * openblack is licensed under the GNU General Public License version 3.
+ *******************************************************************************/
+
+#define LOCATOR_IMPLEMENTATIONS
```

Comment:
You're not checking for this because you're not using an abstract base class. In this case it's not needed.